### PR TITLE
ag: fixed misleading information

### PR DESCRIPTION
**Pull request by NightMachinary**
Fixed a misleading description that wasted quite a bit of my time.

----

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/ag.md [hunk 1]
```diff
@@ -2,26 +2,26 @@
 
 > The Silver Searcher. Like ack, but faster.
 
-- Find files containing "foo", and print the line matches in context:
+- Find files containing "foo", and print the line matches in context (`-F` is an alias for `--literal`):
```
Comment (sbrl) [inline pages/common/ag.md]: What does `--literal` do?

Comment (mfrw) [reply pages/common/ag.md]: @sbrl As per the documentation `Don't parse PATTERN as a regular expression`

Comment (agnivade) [reply pages/common/ag.md]: I would like to keep the first example without any flags. We should change the description instead.

Comment (agnivade) [reply pages/common/ag.md]: Also, maybe bring up the penultimate example to the top. I think that might clarify that foo may be a normal text as well as a regular expression.

**Pull request closed without merging**